v2/cmd/integration-test/interactsh.go

```diff
@@ -1,10 +1,8 @@
 package main
 
-import "github.com/projectdiscovery/nuclei/v2/pkg/testutils"
-
 // All Interactsh related testcases
-var interactshTestCases = map[string]testutils.TestCase{
-	"http/interactsh.yaml":                     &httpInteractshRequest{},
-	"http/interactsh-stop-at-first-match.yaml": &httpInteractshStopAtFirstMatchRequest{},
-	"http/default-matcher-condition.yaml":      &httpDefaultMatcherCondition{},
+var interactshTestCases = []TestCaseInfo{
+	{DisableOn: []string{"windows", "darwin"}, Path: "http/interactsh.yaml", TestCase: &httpInteractshRequest{}},
```

Comment:
As per the previous comment I would change this to:
```go
func() bool { return osutil.IsWindows() || osutil.IsDarwin() }
```
and move it as last item of the list for readability